distribution/docs/src/main/jdocs/content/_configuring/configuring-web-service-providers-contents.adoc

```diff
@@ -0,0 +1,12 @@
+:title: Configuring Web Service Providers
+:type: configuringConfigFile
+:status: published
+:summary: Configuring Web Service Providers.
```

Comment:
I squashed the commit so I can't find the comment, but I think someone suggested that Web Service Providers should be capitalized.